### PR TITLE
Use the `reasonText` atriburte of the extension reccomendation in aria-label

### LINKED ISSUE #51471: airia-label for reccomended extensions contains "[object Object]" inseated of reccomendation text.
Issue Type: <b>Bug</b>

Navigate to any reccomended extension with a screen reader. The reader will read the extension name, followed by "[object Object]", rather than showing that the extension has been reccomended and why.

VS Code version: Code 1.24.0 (6a6e02cef0f2122ee1469765b704faf5d0e0d859, 2018-06-06T17:35:40.560Z)
OS version: Windows_NT x64 10.0.16299

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz (8 x 2112)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: disabled_software<br>rasterization: disabled_software<br>video_decode: enabled<br>video_encode: enabled<br>vpx_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Memory (System)|15.93GB (7.92GB free)|
|Process Argv|C:\Program Files\Microsoft VS Code\Code.exe|
|Screen Reader|no|
|VM|0%|

</details>Extensions: none

### PR DESCRIPTION
**Pull request by JacksonKearl**
Fixes #51471.

### CODE REVIEW

**Comment by JacksonKearl** [comment]
This makes:
```
aria-label="ESLint. This extension is recommended by users of the current workspace."
title="This extension is recommended by users of the current workspace."
```

rather than:
```
aria-label="ESLint. [object Object]"
title="This extension is recommended by users of the current workspace."
```

People on [this stackoverflow thread](https://stackoverflow.com/questions/27953425/what-is-the-difference-between-aria-label-and-title-attributes) seem to say duplicating `title` in `aria-label` is okay (?).

**Review by ramya-rao-a — APPROVED** [review]
Nice catch!

File: src/vs/workbench/parts/extensions/browser/extensionsList.ts [hunk 1]
```diff
@@ -165,7 +165,7 @@ export class Renderer implements IPagedRenderer<IExtension, ITemplateData> {
 
 		const extRecommendations = this.extensionTipsService.getAllRecommendationsWithReason();
 		if (extRecommendations[extension.id.toLowerCase()]) {
-			data.root.setAttribute('aria-label', extension.displayName + '. ' + extRecommendations[extension.id]);
+			data.root.setAttribute('aria-label', extension.displayName + '. ' + extRecommendations[extension.id].reasonText);
```
Comment (ramya-rao-a) [inline src/vs/workbench/parts/extensions/browser/extensionsList.ts]: we should probably add a `.toLowerCase()` for the `extension.id`. That is what the `if` looks for in line 168

Comment (ramya-rao-a) [inline src/vs/workbench/parts/extensions/browser/extensionsList.ts]: we should probably add a `.toLowerCase()` for the `extension.id`. That is what the `if` looks for in line 168

**Review by ramya-rao-a — CHANGES_REQUESTED** [review]

**Review by ramya-rao-a — APPROVED** [review]

**Pull request merged**